src/Bicep.LangServer/Bicep.LangServer.csproj

```diff
@@ -14,4 +14,10 @@
     <PackageReference Include="OmniSharp.Extensions.LanguageServer" Version="0.18.3" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Include="Snippets\Templates\*.bicep">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
>Always [](start = 29, length = 6)

This causes rebuilds in VS. Can you set to `PreserveNewest`?

Reply:
Updated as part of below commit. Thanks!

https://github.com/Azure/bicep/commit/91ec6ff34132d0b6be98410a28548453a5b15375